src/shadowbox/integration_test/test.sh

```diff
@@ -63,8 +63,9 @@ function ss_arguments_for_user() {
 }
 
 # Runs curl on the client container.
+readonly CURL_OUTPUT=$(mktemp)
 function client_curl() {
-  docker exec $CLIENT_CONTAINER curl --silent --show-error "$@"
+  docker exec $CLIENT_CONTAINER curl --silent --show-error "$@" | tee $CURL_OUTPUT
```

Comment:
Is this extra output for debugging? In any case, you need to return curl's exit status (as opposed to tee's) for the tests to pass.

Reply:
yup, that's what I'm noticing as well.  At this point this branch is now for testing on Travis as it's the only place I can get the flake to reproduce

Reply:
Remove the `| tee` call. It depends on a global and creates a hidden dependency. You can change callers instead when you need.